pydantic/generics.py

```diff
@@ -241,3 +236,27 @@ def get_caller_frame_info() -> Tuple[Optional[str], bool]:
         return None, False
     frame_globals = previous_caller_frame.f_globals
     return frame_globals.get('__name__'), previous_caller_frame.f_locals is frame_globals
+
+
+def _prepare_model_fields(
+    created_model: Type[GenericModel],
+    fields: Mapping[str, Any],
+    instance_type_hints: Mapping[str, type],
+    typevars_map: Mapping[Any, type],
+) -> None:
+    """Replace DeferredType fields w/ concrete type hints and prepare them."""
```

Comment:
```suggestion
    """
    Replace DeferredType fields with concrete type hints and prepare them.
    """
```

sorry, this is just a personal tick of mine.